docs/announcements/github-release-v0.1.84.md

```
@@ -0,0 +1,23 @@
+# 🚀 Release Highlights — v0.1.84 (2026-05-08)
+
+### 🗺️ [TUI Consensus Map](https://docs.massgen.ai/en/latest/user_guide/tui.html)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)